rancher-webhook.yaml

```diff
@@ -1,10 +1,13 @@
 package:
-  name: rancher-webhook
+  name: rancher-webhook-0.5
```

Comment:
I think we'll need a tag filter on the update config so the update bot tracks 0.5 releases?

Reply:
yep for sure, I was saying that I'm missing smth 🙈